### PR TITLE
Next Version

### PR DESCRIPTION
**Pull request by donavanbecker**

### CODE REVIEW

**Comment by donavanbecker** [comment]
@CocoaBob We can use this to discuss when this is ready.

**Comment by CocoaBob** [comment]
@donavanbecker During the merging, the following part of codes wasn't deleted, they need to be deleted 😉
```
// Stop updating after 22 seconds
self.checkStateTimeout = setTimeout(function () {
  self.stopRequestingDoorState();
}, this.config.garageDoorOpeningTime * 1000 + 2000);
```
Since the method `stopRequestingDoorState()` has been renamed to `stopUpdatingDoorState()`, Homebridge is crashing every 2 minutes now.
image

**Comment by donavanbecker** [comment]
> Please remove these lines
> 
> ```
> // Stop updating after 22 seconds
> self.checkStateTimeout = setTimeout(function () {
>     self.stopRequestingDoorState();
> }, this.config.garageDoorOpeningTime * 1000 + 2000);
> ```

Okay done. Wanna check the next beta? should be published.

File: index.js [hunk 1]
```diff
@@ -561,32 +554,20 @@ class Meross {
...
-      .catch(function(error) {
-        self.stopRequestingDoorState();
-      });
+      .catch( e => this.log.debug(`${e}`));
     }, 2000);
-    // Stop updating after 22 seconds
-    self.checkStateTimeout = setTimeout(function () {
-      self.stopRequestingDoorState();
-    }, this.config.garageDoorOpeningTime * 1000 + 2000);
-  }
```
Comment (CocoaBob) [inline index.js:584]: We still need this bracket `}`

Comment (CocoaBob) [reply index.js:584]: I meant, the last one.

Comment (donavanbecker) [reply index.js:584]: Should be good now?

**Comment by CocoaBob** [comment]
Now it works well! Just like what I've been testing on my branch, thanks!

**Pull request merged**